src/mutation/RelayMutation.js

```diff
@@ -50,12 +50,29 @@ class RelayMutation<Tp: Object> {
     route: RelayMetaRoute
   ) => Variables;
 
-  props: Tp;
+  _context: RelayContext;
   _didShowFakeDataWarning: boolean;
+  _unresolvedProps: Tp;
+  props: Tp;
 
   constructor(props: Tp) {
-    this._didShowFakeDataWarning = false;
-    this._resolveProps(props);
+    this._unresolvedProps = props;
+  }
+
+  /**
+   * @internal
+   */
+  bindContext(context: RelayContext): void {
+    if (!this._context) {
+      this._context = context;
+      this._resolveProps();
```

Comment:
Here you go: https://github.com/denvned/codemod/blob/master/transforms/RelayMutation-didResolveProps.js

I tried to make it as safe as possible, consequently it makes transformation only if `super(props)` is the first statement in the constructor (I hope it is the most common case, please tell if it is not). There are also other safety checks. And it reports the reason for every skipped case.

The first codemod I ever wrote, writing it was exciting :neckbeard: